FIAT/bernardi_raugel.py

```diff
@@ -0,0 +1,85 @@
+# This file is part of FIAT (https://www.fenicsproject.org)
+#
+# SPDX-License-Identifier:    LGPL-3.0-or-later
+#
+# Written by Pablo D. Brubeck (brubeck@protonmail.com), 2024
+
+from FIAT import finite_element, dual_set, polynomial_set, expansions
+from FIAT.functional import ComponentPointEvaluation, FrobeniusIntegralMoment
+from FIAT.quadrature_schemes import create_quadrature
+from FIAT.quadrature import FacetQuadratureRule
+from itertools import chain
+import numpy
+
+
+def ExtendedBernardiRaugelSpace(ref_el, degree):
+    """Return a basis for the extended Bernardi-Raugel space.
+    P_1^d + (P_{d} - P_{d-1})^d"""
```

Comment:
is P_d - P_{d-1} literately anything that is the difference between a P_d and a P_{d-1} thing?  That would just be P_d, generically.  Do you mean some kind of homogeneous space?

Reply:
I wanted to say `P_d \ P_{d-1}`, refering to a homogenous space, but lint complained about the backspace

Reply:
Then update the comment to just say homogeneous polynomials or some other wording?  (Maybe it would take a double backslash?)